components/status_icon.tsx

```diff
@@ -60,6 +60,6 @@ export default class StatusIcon extends React.PureComponent {
             IconComponent = StatusOfflineIcon;
         }
 
-        return <IconComponent className={className}/>;
+        return <IconComponent {...iconComponentProps}/>;
```

Comment:
I'm passing the props with the spread operator like this to fix the following TS error on ```<IconComponent className={className}/>``` 

> TS2322: Type '{ className: string; }' is not assignable to type 'IntrinsicAttributes & IntrinsicClassAttributes<Component<{}, any, any>> & Readonly<{}> & Readonly<{ children?: ReactNode; }>'.   Property 'className' does not exist on type 'IntrinsicAttributes & IntrinsicClassAttributes<Component<{}, any, any>> & Readonly<{}> & Readonly<{ children?: ReactNode; }>'.

I found the solution from [this SO post](https://stackoverflow.com/a/57312722
)